src/Gtk/SnapshotBackendBox.vala

```diff
@@ -212,6 +232,7 @@ class SnapshotBackendBox : Gtk.Box{
 	public void refresh(){
 		
 		opt_btrfs.active = App.btrfs_mode;
+		opt_btrfs_readonly.active = App.btrfs_readonly;
```

Comment:
This will need to be guarded by `if (opt_btrfs.active) {}` or else it throws a runtime warning. What I'd prefer is to always add the widget, but mark it insensitive (grey it out) - like we already do for the btrfs radio button:
https://github.com/linuxmint/timeshift/pull/373/files#diff-4a9255872e366bab354c5ddc6d435d03d84ae8b736856a284092dc137ff21257R101-R104